lib/init.ts

```diff
@@ -1,5 +1,4 @@
-import { isUndefined, isString } from "./utils/index";
-import { isNumber } from "util";
```

Comment:
Which means that we have `util` in the current build of the lib?

Reply:
let's add bundlesize to avoid this in the future!

Reply:
🤦‍♂️ my mistake.
No it's not in the bundle. rollup doesn't include node builtins unless you tell it to 
https://www.npmjs.com/package/rollup-plugin-node-builtins

Reply:
Actually you're right. It was included in the bundle 😱 +3kB